questions/qa-personal-names.en.html

```diff
@@ -262,13 +263,12 @@ <h3 id="otherthings"><a href="#otherthings">Other things</a></h3>
       <mark id="req_spaces" class="requirement">Allow the user to enter a name with spaces</mark>
       , eg. to support prefixes and suffixes
       such as de in French, von in German, and Jnr/Jr  in American names, and also because some people consider a space-separated sequence of characters to be a single name, eg. Rose Marie.</p>
-    <p>
-      <mark id="req_samefamily" class="requirement">Don't assume that members of the same family will share the same family name.</mark>
-    </p>
-    <p>
-<mark id="req_title" class="requirement">Ask yourself whether it's necessary to require that people supply a title.</mark> 
+<p><mark id="req_case_normalization">Don't normalize the casing in names.</mark>
```

Comment:
This is very useful. Maybe it can be added to [specdev](https://w3c.github.io/bp-i18n-specdev/#loc_names).

Reply:
You're very welcome to add it :-)

Reply:
See https://github.com/w3c/bp-i18n-specdev/pull/47